scripts/test-react.js

```diff
@@ -682,6 +682,23 @@ function runTestSuite(outputJsx, shouldTranspileSource) {
       it("Dynamic ReactElement type #3", async () => {
         await runTest(directory, "dynamic-type3.js");
       });
+
+      it("Lazy branched elements", async () => {
+        let createElement = React.createElement;
+        let count = 0;
+        // $FlowFixMe: intentional for this test
+        React.createElement = (type, config) => {
+          count++;
+          return createElement(type, config);
+        };
+        try {
+          await runTest(directory, "lazy-branched-elements.js");
+        } finally {
+          // $FlowFixMe: intentional for this test
+          React.createElement = createElement;
+        }
+        expect(count).toEqual(8);
```

Comment:
Can you add a clarifying comment? Like

```js
        // App + span on first render, App + div on second render. (= 4)
        // Repeated twice (= 8) because both original and Prepacked code does it.
```